transformations/transformations/single_column_coalesced.py

```diff
@@ -308,27 +288,10 @@ def process_kernel(self, routine):
         # Find the iteration index variable for the specified horizontal
         v_index = self.get_integer_variable(routine, name=self.horizontal.index)
 
-        # Transform arrays passed with scalar syntax to array syntax
-        if self.resolve_sequence_association:
-            transform_sequence_association(routine)
-
-        # Perform full source-inlining for member subroutines
-        if self.inline_members:
-            inline_member_procedures(routine, allowed_aliases=(self.horizontal.index,))
-
-        # Perform full source-inlining for pragma-marked subroutines
-        if self.inline_marked:
-            # When inlining we allow the horizontal dimension to alias, so that
-            # the de/re-vectorisation captures the shared vector dimension.
-            inline_marked_subroutines(routine, allowed_aliases=(self.horizontal.index,))
-
         # Associates at the highest level, so they don't interfere
         # with the sections we need to do for detecting subroutine calls
         resolve_associates(routine)
```

Comment:
What's the reason for not including `resolve_associates` in the `SanitiseTransformation`?

Reply:
Oversight, now fixed (I hope).

Reply:
Sure, I only asked for the reason, and that's a good enough one ;-)